package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "nsfw",
-  "version": "1.0.16",
+  "name": "@atom/nsfw",
```

Comment:
I'm guessing this was unintentional

Reply:
Oh yeah, that shouldn't be in this PR. Thanks!